republish_invoices/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/getAlby/lndhub.go/db"
+	"github.com/getAlby/lndhub.go/db/models"
+	"github.com/getAlby/lndhub.go/lib/service"
+	"github.com/getAlby/lndhub.go/rabbitmq"
+	"github.com/joho/godotenv"
+	"github.com/kelseyhightower/envconfig"
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+
+	c := &service.Config{}
+	// Load configruation from environment variables
+	err := godotenv.Load(".env")
+	if err != nil {
+		fmt.Println("Failed to load .env file")
+	}
+	startDate, endDate, err := loadStartAndEndIdFromEnv()
+	if err != nil {
+		log.Fatalf("Could not load start and end id from env %v", err)
+	}
+	err = envconfig.Process("", c)
+	if err != nil {
+		log.Fatalf("Error loading environment variables: %v", err)
+	}
+	// Open a DB connection based on the configured DATABASE_URI
+	dbConn, err := db.Open(c)
+	if err != nil {
+		log.Fatalf("Error initializing db connection: %v", err)
+	}
+	rabbitmqClient, err := rabbitmq.Dial(c.RabbitMQUri,
+		rabbitmq.WithLndInvoiceExchange(c.RabbitMQLndInvoiceExchange),
+		rabbitmq.WithLndHubInvoiceExchange(c.RabbitMQLndhubInvoiceExchange),
+		rabbitmq.WithLndInvoiceConsumerQueueName(c.RabbitMQInvoiceConsumerQueueName),
+	)
```

Comment:
should this be a function like `getRabbitmqClient` to make sure we have the same client config everwhere?